src/Ushahidi/Modules/V5/Http/Controllers/RoleController.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Http\Controllers;
+
+use App\Bus\Query\QueryBus;
+use App\Bus\Command\CommandBus;
+use Ushahidi\Modules\V5\Http\Resources\Role\RoleCollection;
+use Ushahidi\Modules\V5\Http\Resources\Role\RoleResource;
+use Illuminate\Http\Request;
+use Ushahidi\Modules\V5\Models\Role;
+use Ushahidi\Modules\V5\Actions\Role\Queries\FetchRoleByIdQuery;
+use Ushahidi\Modules\V5\Actions\Role\Queries\FetchRoleQuery;
+use Ushahidi\Modules\V5\Requests\StoreRoleRequest;
+use Ushahidi\Modules\V5\Actions\Role\Commands\CreateRoleCommand;
+use Ushahidi\Modules\V5\Actions\Role\Commands\DeleteRoleCommand;
+use Ushahidi\Modules\V5\Actions\Role\Commands\UpdateRoleCommand;
+use Ushahidi\Core\Exception\AuthorizerException;
+use Illuminate\Auth\Access\AuthorizationException;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Contracts\Auth\Access\Gate;
+use Ushahidi\Modules\V5\Actions\Role\Commands\CreateRolePermissionCommand;
+use Ushahidi\Modules\V5\Actions\Role\Commands\DeleteRolePermissionByRoleCommand;
+
+class RoleController extends V5Controller
+{
+
+
```

Comment:
```suggestion
private $queryBus;
privaet $commandBus;

public function __construct(QueryBus $queryBus, CommandBus $commandBus)
{
    $this->queryBus = $queryBus;
    $this->commandBus = $commandBus;
}
```

This way you won't need to pass QueryBus and CommandBus to methods (and it's tiny bit faster as Laravel resolves it quicker)